images/plbase/python-requirements.txt

```diff
@@ -13,11 +13,11 @@ pandas-stubs==2.2.3.241126
 pandas==2.2.3
 Pillow==11.1.0
 Pint==0.24.4
-psutil==6.1.1
-PuLP==2.9.0
+psutil==7.0.0
```

Comment:
This appears to mostly be dropping Python 2.7 support: https://github.com/giampaolo/psutil/blob/master/HISTORY.rst